test/Tests/src/Data/Ordering/Comparator_Spec.enso

```diff
@@ -13,12 +13,19 @@ import Standard.Test.Extensions
 type Ord
     Value number
 
-    compare_to : Ord -> Ordering
-    compare_to self that = that.number.compare_to self.number
+# The comparison is reverted, i.e., `x < y` gives result for `y.number < x.number`.
+type Ord_Comparator
+    is_ordered = True
+    compare x y = (Comparable.from y.number) . compare y.number x.number
+    hash x = (Comparable.from x.number) . hash x.number
+
+Comparable.from (_:Ord) = Ord_Comparator
 
 type No_Ord
     Value number
 
+Comparable.from (_:No_Ord) = Incomparable
+
```

Comment:
Because by default, there is `Comparable.from (_:Any) = Default_Unordered_Comparator`, so that we can check anything with anything for equality. This line means that `x == y` will be `False` for any `No_Ord` object.